cadquery/cq.py

```diff
@@ -1312,14 +1312,16 @@ def move(self, xDist=0, yDist=0):
         return self.newObject([self.plane.toWorldCoords(newCenter)])
 
     def spline(self, listOfXYTuple, tangents=None, periodic=False,
-               forConstruction=False):
+               forConstruction=False, includeCurrent=True, makeWire=False):
```

Comment:
I'm thinking the default values of the two new parameters have the correct defaults to make sure spline works as it always has. If so, I think this change is fine.

Reply:
Indeed. Spline tests pass without any modifications.